backend/bom-bom-server/src/main/java/me/bombom/api/v1/article/repository/ArticleRepositoryImpl.java

```diff
@@ -77,6 +78,17 @@ public int countAllByCategoryIdAndMemberId(Long memberId, Long categoryId, Strin
                 .intValue();
     }
 
+    @Override
+    public int countByMemberIdAndArrivedDateTimeAndIsRead(Long memberId, LocalDate date, boolean isRead) {
+        return jpaQueryFactory.select(article.count())
+                .from(article)
+                .where(createMemberWhereClause(memberId))
+                .where(createDateWhereClause(date))
+                .where(article.isRead.eq(isRead))
+                .fetchOne()
+                .intValue();
+    }
```

Comment:
p2

`fetchOne()`은  null을 반환할 수 있기 때문에, `intValue()` 호출 시 NPE이 발생할 가능성이 있어요
그래서 아래처럼 null을 방어해주면 더 좋을 것 같아요 😊

```suggestion
    public int countByMemberIdAndArrivedDateTimeAndIsRead(Long memberId, LocalDate date, boolean isRead) {
        Long count = jpaQueryFactory.select(article.count())
        .from(article)
        .where(createMemberWhereClause(memberId))
        .where(createDateWhereClause(date))
        .where(article.isRead.eq(isRead))
        .fetchOne();

return Optional.ofNullable(count)
        .orElse(0L)
        .intValue();
```